code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -48,7 +48,7 @@
 		if(14)
 			new /obj/item/nullrod/scythe/talking(src)
 		if(15)
-			new /obj/item/nullrod/armblade(src)
+			new /obj/item/changestone(src)
```

Comment:
This isnt a fix. This is an alternative.

I also slated for this thing to be removed. Its not accessible and was added before para even existed.
![image](https://user-images.githubusercontent.com/25063394/121735452-0df32d80-caee-11eb-8de9-ab475940e2ce.png)


If you're going to make a "fix", **actually fix this fucking problem**